internal/store/serverservice.go

```diff
@@ -179,6 +180,32 @@ func (s *Serverservice) Create(ctx context.Context, serverID uuid.UUID, createCo
 	return err
 }
 
+// AddServer creates a server record in FleetDB
+// @id: required
+// @ip: required
+// @user: required
+// @pwd: required
+func (s *Serverservice) AddServer(ctx context.Context, serverID uuid.UUID, bmcAddr, bmcUser, bmcPass string) error {
```

Comment:
This needs to accept the facility code parameter as well, each server must have a facility code.